trunk/src/datstrs/get_index.F90

```diff
@@ -127,14 +127,7 @@ subroutine get_index(Nod, Indexd)
 !              ...loop through components
                   do ivar=1,NR_COMP(iphys)
                      ic=ic+1
-                     select case(ibcd(ic))
-!
-!                    ...free H(div) component
-                        case(0); Indexd(ic)=8
-!
-!                    ...component known from Dirichlet BC
-                        case(1); Indexd(ic)=7
-                     end select
+                     Indexd(ic)=8
```

Comment:
I think we can make this change, however we should then also adapt `celem_system` accordingly. Currently, `celem_system` implements `case(7)` for `L2` components which perhaps was previously used for removing `L2` components from the system in some assembly/solver hack. Otherwise, I'm not sure about the use case there.